static/src/stylesheets/module/experiments/_embed.scss

```diff
@@ -0,0 +1,171 @@
+@import 'svg';
+
+
+$contribute-yellow: #ffbb00;
+$gu-blue: #005689;
+$gu-blue-dark: #003d70;
+$comment-support-1: #7d7569;
+$white: #ffffff;
+$black: #000000;
+$gs-gutter: 20px;
+
+
+
+.interactive-embed {
+    height: 100%;
+    width: 100%;
+    border-top: 1px solid #4bc6df;
+    background-color: #efefec;
+
+    p {
```

Comment:
We generally try and style on classes - I would try and add classes to your elements with the BEM element syntax `.interactive-embed__subtitle` and not nest at all